docs/TuxMutableClass.md

```diff
@@ -0,0 +1,188 @@
+
+These `mutableTraits` paths are then used in the `shouldComponentUpdate` lifecycle event to see if those properties have changed when comparing the current state and props to the next props and state of the component. If those state and/or props have not changed then `shouldComponentUpdate` will return false which indicates to React that it does not need to update the component in the DOM. By performing this check you can prevent the unneeded re-rendering of components within the virtual DOM each time a potential re-render is triggered.
+
+The mutable class in `Tux` is an extension of the `OwneeClass`, which is an implementation on top of React's `createClass` method. Which means that it has access to all of the same [lifecycle](http://facebook.github.io/react/docs/component-specs.html) methods as a normal React class. Additionally, it requires a `render` method just like any other React component.
+
+### <a id="Requiring-TuxMutableClass"></a>1) Requiring Tux Mutable Class [#](#Requiring-TuxMutableClass)
+The Tux Mutable Class method `createMutableClass` is exposed by requiring the Tux React module, which can be required via:
+
+```javascript
+  var React = require('tux/React');
+```
+
+**Note that at this time for Tux to be compiled by the JSX transformer, the Tux module must be assigned to the variable name `React`**
+
+### <a id="creating-mutable-class-components"></a>2) Creating Mutable Class Components [#](#creating-mutable-class-components)
+The Tux extended `React` object allows users to create mutable class components via invoking [React.createMutableClass](#createMutableClass). Pass in an object to the `createMutableClass` method and it will return a `TuxMutableClass` which is an extension of a generic React Class.
+
+Let's take a look at an example: [#](#create-mutable-class-example)
+<a id="create-mutable-class-example"></a>
+
+```javascript
+  var Message = React.createMutableClass({
+    // the traits we want to watch to see if a re-render is required
+    mutableTraits: {
+      props: ['text', 'timestamp'],
+      state: 'editing'
+    },
+
+    render: function () {
+      return (
+        <div>
+          <h1>{this.props.message.text}</h1>
+        </div>
+      );
+    }
+  });
+```
+
+Below is the api documentation for this method.
+
+### <a id="React-createMutableClass"></a>React.createMutableClass [#](#React-createMutableClass)
+
+```javascript
+  var Message = React.createMutableClass(mutableClassProps);
+```
+
+#### <a id="createMutableClass-mutableClassProps"></a>Parameter - `mutableClassProps` - type: OBJECT - required [#](#createMutableClass-mutableClassProps)
+Methods and properties to be extended onto the `mutableClass` object returned. Expected keys:
```

Comment:
Change to Methods and properties to extend the returned `mutableClass` object with.